chain-signatures/contract/tests/tests.rs

```diff
@@ -499,18 +523,15 @@ async fn test_propose_update_config(contract: &Contract, accounts: &[Account]) {
     assert_eq!(config, new_config);
 }
 
-async fn _test_propose_update_contract(contract: &Contract, accounts: &[Account]) {
-    const CONTRACT_DEPLOY: NearToken = NearToken::from_near(8);
+async fn test_propose_update_contract(contract: &Contract, accounts: &[Account]) {
+    const CONTRACT_DEPLOY: NearToken = NearToken::from_millinear(8500);
     let state: mpc_contract::ProtocolContractState =
         contract.view("state").await.unwrap().json().unwrap();
 
     // Let's propose a contract update instead now.
-    let new_wasm = std::fs::read(CONTRACT_FILE_PATH).unwrap();
     let execution = accounts[0]
         .call(contract.id(), "propose_update")
-        .args_json(serde_json::json!({
-            "code": &new_wasm,
-        }))
+        .args_borsh((current_contract(),))
```

Comment:
@ChaoticTempest how do additional brackets help here? :)

Reply:
Borsh serialization expects an iterable list of args. Adding `(T,)` makes it into a tuple type that is iterable